source/refined-github.css

```
@@ -116,3 +116,8 @@ pr-branches
 .js-issue-row .text-small.text-gray {
 	line-height: 1.8;
 }
+
+/* Reset `vertical-align` for .octicons to work with above increased line-height */
+.js-issue-row .text-small.text-gray .octicon {
+	vertical-align: middle !important;
+}
```

Let's match GitHub’s own selector for the alignment. Also `!important` doesn't seem to be needed. Can you double-check? 

```suggestion
.issue-meta-section .octicon {
	vertical-align: middle;
}
```

---

Your suggestion makes sense in my books, `stylelint` disagrees :stuck_out_tongue:. OK if I disable it for the line?

---

I just saw why you chose that selector in the first place. Feel free to restore `.js-issue-row .text-small.text-gray .octicon` to fix the issue :D

---

Ok, reverted it but removed `important` as it was indeed not needed. Should be all good now